bot.py

```diff
@@ -591,6 +606,15 @@ def main_color(self) -> int:
             logger.warning("Error with 'guild_age'.", exc_info=True)
             self.config.remove("guild_age")
 
+        try:
+            last_log = (await self.api.get_user_logs(message.author.id))[-1]
+            cooldown = datetime.strptime(last_log["created_at"].split(".")[0], '%Y-%m-%d %H:%M:%S') + isodate.parse_duration(self.config["thread_cooldown"])
```

Comment:
`isodate.parse_duration(self.config["thread_cooldown"])` should be replaced with `thread_cooldown`. and parse created_at with `https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat`.